providers/base/units/dock/jobs.pxu

```diff
@@ -2355,85 +2355,535 @@ _description: Check dock MAC address pass-through function, please make sure fol
-     2. Waiting 30 seconds.
-     3. Resume the system.
+    1. Suspend the system.
+    2. Waiting 30 seconds.
+    3. Resume the system.
 _verification:
-     1. Did the display works fine after suspend?
-     2. Did the storage works fine after suspend?
-     3. Did the audio works fine after suspend?
-     4. Did the network works fine after suspend?
+    Mark it Pass if you have suspended and resumed the system.
 
 id: dock/all-poweroff
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after poweroff
+    This test is to check if the device is still working after power-off.
 _steps:
-     1. Poweroff the system.
-     2. Waiting 30 seconds.
-     3. Poweron the system.
+    1. Poweroff the system.
+    2. Wait for 30 seconds.
+    3. Poweron the system.
 _verification:
-     1. Did the display works fine after poweroff?
-     2. Did the storage works fine after poweroff?
-     3. Did the audio works fine after poweroff?
-     4. Did the network works fine after poweroff?
+    Mark it Pass if you have powered on and powered off the system.
+
+id: dock/all-init-monitor-multi-head
+category_id: dock-display
+estimated_duration: 120.0
+_summary: Dual monitors test while docked
+plugin: manual
+_purpose:
+    This test is to verify that multi-monitor output works using the dock. You will need multiple external monitors to perform this test. Depends on the specification of the device to use 4K monitor or FHD monitor.
+_steps:
+    Skip this test if your video card or the dock do not support multiple monitors.
+    1. If your dock provides more than one monitor outputs, connect multiple monitors.
+    2. Open the "Displays" tool (open the dash and search for "Displays").
+    3. Configure your output to provide one desktop across all the monitors.
+    4. Open any application and drag its window from one monitor to the next.
+_verification:
+    Was the stretched desktop displayed correctly across all the screens?
+_siblings:
+    [{"id": "dock/all-reboot-monitor-multi-head"},
+     {"id": "dock/all-hotplug-monitor-multi-head"},
+     {"id": "dock/all-suspend-monitor-multi-head"},
+     {"id": "dock/all-poweroff-monitor-multi-head"}]
+
+id: dock/all-init-monitor-multi-head-audio-playback
+category_id: dock-audio
+requires:
+    device.category == 'AUDIO'
+_summary: Multiple monitor audio test
+plugin: user-interact-verify
+estimated_duration: 60.0
+command:
+    indexes=$(pacmd list-sinks | grep -e 'index' -e 'available' | grep -B 1 -e 'available: unknown' -e 'available: yes' | grep index | awk '{print $NF}')
+    for index in $indexes
+    do
+            pacmd set-default-sink "$index"
+            audio_settings.py store --file="$PLAINBOX_SESSION_SHARE"/dockaudio_settings
+            pacmd set-sink-volume "$index" 65536
+            speaker-test -c 2 -l 1 -t wav
+            audio_settings.py restore --file="$PLAINBOX_SESSION_SHARE"/dockaudio_settings
+    done
```

Comment:
This might benefit from being a separate shell script in the `bin` directory.

Reply:
These command also use some python scripts from `bin/`, mabe it's not a good idea write these into a shell script in `bin/`?

Reply:
I'm not sure to follow your reasoning. You can create a shell script in `bin/` that calls a python script in `bin/`... no?

Reply:
Yeah, you are right. Of course I could create a shell script in `bin/` that calls a python script in `bin/`. I am just wondering if it would be too complicated? Like [this command](https://github.com/canonical/checkbox/blob/f9a045d55eb5d36c581720fddb97a110a5f10c44/providers/base/units/audio/jobs.pxu#L76) and any other command in `audio test` are not written in a shell script. I think it's because the `command` in every `job`s still have slightly differences and there would be too much binary file being created if we write every `command`s into scripst. These could explain why I didn't make these command into shell scripts in the discussion below.